test/transpiler/transpiler_array_test.ts

```diff
@@ -122,23 +122,36 @@ describe('Transpiler', () => {
             wrapper.setFunctionName('arrayExport');
 
             const array1 = [0];
-            wrapper.call(array1);
+            wrapper.setOutParameters(array1).call(array1);
             expect(array1[0]).to.equal(5);
 
             const array2 = [1, 2, 3];
-            wrapper.call(array2);
+            wrapper.setOutParameters(array2).call(array2);
             expect(array2[0]).to.equal(5);
             expect(array2[1]).to.equal(2);
             expect(array2[2]).to.equal(3);
         });
 
+        it('should handle array export with non exportable values', () => {
+            const wrapper = transpiler.transpile('function arrayExport(arr, value) { arr[0] = value; return arr[0]; }');
+            wrapper.setFunctionName('arrayExport');
+
+            const array = [0];
+            wrapper.setOutParameters(array).call(array, 3);
+            expect(array[0]).to.equal(3);
+        });
+
         it('should handle multiple array wrapper', () => {
```

Comment:
What do you mean by wrapper here?

Reply:
That accidentally happened because of a search & replace.